tests/python/quantization/test_quantization.py

```diff
@@ -229,69 +219,106 @@ def check_quantized_conv(data_shape, kernel, num_filter, pad, stride, dilate, no
+                    'pad': padding, 'num_filter': channels, 'no_bias': not use_bias, 'num_group': 1,
+                    'layout': 'NCHW'}
+
+                self.min_data = mx.gluon.Parameter('min_data', dtype='float32', allow_deferred_init=True)
+                self.max_data = mx.gluon.Parameter('max_data', dtype='float32', allow_deferred_init=True)
+
+                self.weight = mx.gluon.Parameter('weight', dtype='int8', allow_deferred_init=True)
+                self.min_weight = mx.gluon.Parameter('min_weight', dtype='float32', allow_deferred_init=True)
+                self.max_weight = mx.gluon.Parameter('max_weight', dtype='float32', allow_deferred_init=True)
+                
+                if use_bias:
+                    self.bias = mx.gluon.Parameter('bias', dtype='int8', allow_deferred_init=True)
+                    self.min_bias = mx.gluon.Parameter('min_bias', dtype='float32', allow_deferred_init=True)
+                    self.max_bias = mx.gluon.Parameter('max_bias', dtype='float32', allow_deferred_init=True)
+
+            def hybrid_forward(self, F, x, weight, bias=None, min_data=None, max_data=None,
+                               min_weight=None, max_weight=None, min_bias=None, max_bias=None):
+                out = F.contrib.quantized_conv(data=x, weight=weight, bias=bias, 
+                                               min_data=min_data, max_data=max_data,
+                                               min_weight=min_weight, max_weight=max_weight,
+                                               min_bias=min_bias, max_bias=max_bias,
+                                               **self._kwargs)
+                return out
+
+        convint8 = QuantConv(channels=num_filter, kernel_size=kernel, strides=stride,
+                             padding=pad, dilation=dilate, use_bias=use_bias)
+
         quantized_range = 127.0
-        if no_bias:
-            conv_exe_int8.arg_dict[qarg_names[2]][:] = -quantized_range
-            conv_exe_int8.arg_dict[qarg_names[3]][:] = quantized_range
-            conv_exe_int8.arg_dict[qarg_names[4]][:] = -quantized_range
-            conv_exe_int8.arg_dict[qarg_names[5]][:] = quantized_range
-        else:
-            conv_exe_int8.arg_dict[qarg_names[2]][:] = conv_exe_fp32.arg_dict[arg_names[2]].astype('int8')
-            conv_exe_int8.arg_dict[qarg_names[3]][:] = -quantized_range
-            conv_exe_int8.arg_dict[qarg_names[4]][:] = quantized_range
-            conv_exe_int8.arg_dict[qarg_names[5]][:] = -quantized_range
-            conv_exe_int8.arg_dict[qarg_names[6]][:] = quantized_range
-            conv_exe_int8.arg_dict[qarg_names[7]][:] = -quantized_range
-            conv_exe_int8.arg_dict[qarg_names[8]][:] = quantized_range
-        qoutput, min_range, max_range = conv_exe_int8.forward()
-
-        if no_bias:
-            assert_almost_equal(output.asnumpy(), qoutput.asnumpy(), atol = 1)
-        else:
+        q_args = {
```

Comment:
Maybe 'qargs' to stay consistent with 'qdtype' and 'qoutput'? Or, as it is more readable with the underscore, change 'qdtype' and 'qoutput' to 'q_dtype' and 'q_output' respectively?